wca-guidelines.md

```diff
@@ -155,10 +155,7 @@ To be more informative, each Guideline is classified using one of the following
 ## <article-A><speedsolving><speedsolving> Article A: Speed Solving
 
 - A1a2+) [ADDITION] In case of a cumulative time limit, the judge records the original recorded time for a DNF on the score sheet in parentheses, e.g. "DNF (1:02.27)".
-- A1a2++) [CLARIFICATION] Cumulative time limits may be enforced for a single round or a combination of rounds of different events.
-- A1a2+++) [CLARIFICATION] In case of a cumulative time limit, the result of each remaining attempt in the round is recorded as DNS (Did Not Start) after a competitor reaches the time limit.
-- A1a2++++) [ADDITION] If a cumulative time limit for a single event or a combination of events is enforced, such events don't have a regular time limit per solve.
-- A1a2+++++) [CLARIFICATION] Events where a time limit per attempt is defined by the Regulations (3x3x3 Fewest Moves and 3x3x3 Multi-Blind) must not have a cumulative time limit.
+- A1a2++) [EXAMPLE] If a round has a cumulative time limit of 30 minutes and the competitor's results first two attempts are 6:20.96 and DNF (10:20), then the time limit for the third attempt is 30:00.00 - 6:20.96 - 10:20 = 13:19.04. For this particular example, note that only the final result for an attempt counts. Therefore, a third attempt with an original recorded time of 13:19.45 without penalties meets the time limit because the final result is 13:19 (see [Regulation 9f2](regulations:regulation:9f2)), while a third attempt with a result of 13:18.00 + 2 = 13:20 does not meet the time limit.
```

Comment:
I would prefer to keep the examples as simple as possible, i.e. `6:00.00` instead of `6:20.96`. I don't want people to lose the point while trying to calculate or trying to remember if times get truncated or rounded.

Reply:
I wanted to add an example that clarifies what happens when you need to mix times with varying precision.

How about having a simple example here, and another example to clarify the mixed precision case?

Reply:
Sounds good to me!